sys/linux/bpf.txt

```diff
@@ -97,11 +97,30 @@ bpf_prog {
 	prog_name		array[const[0, int8], BPF_OBJ_NAME_LEN]
 	prog_ifindex		ifindex[opt]
 	expected_attach_type	flags[bpf_attach_type, int32]
+	btf_fd			fd_btf
+	func_info_rec_size	len[func_info, int32]
+	func_info		ptr64[in, bpf_func_info]
+	func_info_cnt		int32
+	line_info_rec_size	len[line_info, int32]
```

Comment:
If this is structured in the same way as func_info, then the same comments apply here.